cmd/ethrex/utils.rs

```diff
@@ -111,12 +111,6 @@ pub fn default_datadir() -> String {
 
 // TODO: Use PathBuf instead of strings
```

Comment:
Can we leverage this PR to address this?

Reply:
I tried, but if I add this, the PR will balloon in diff size. Better to leave it for later.

Reply:
Refactored in #4543